### PR TITLE
Added proxy lambda to do cross region trigger

### PR DESCRIPTION
**Pull request by jophine-antony**
Added proxy lambda to do cross region trigger

### CODE REVIEW

File: awsconfigrules/build.f/particles-awsconfigrules/particles/cftemplates/template.json.hbs [hunk 1]
```diff
@@ -11,6 +11,7 @@
   {{resource "iam_user_policy_checker_lambda" logicalId="IAMUserPolicyCheckerLambda"}}
   {{resource "sg_portrange_lambda" logicalId="SGPortRangeLambda"}}
   {{resource "ec2_create_trigger_db_update_lambda" logicalId="EC2CreateTriggerDbUpdateLambda"}}
+  {{resource "ec2_create_trigger_db_update_lambda_proxy" logicalId="EC2CreateTriggerDbUpdateLambdaProxy"}}
```
Comment (RajkumarShivage) [inline awsconfigrules/build.f/particles-awsconfigrules/particles/cftemplates/template.json.hbs]: Rename it to ec2_create_trigger_master_lambda_proxy

Comment (RajkumarShivage) [inline awsconfigrules/build.f/particles-awsconfigrules/particles/cftemplates/template.json.hbs]: Rename it to EC2CreateTriggerMasterLambdaProxy

File: awsconfigrules/build.f/particles-awsconfigrules/particles/cftemplates/template.json.hbs [hunk 2]
```diff
@@ -19,4 +20,5 @@
   {{output "fetch_rules_lambda_arn" logicalId="FetchRulesLambdaArn"}}
   {{output "sg_portrange_lambda_arn" logicalId="SGPortRangeLambdaArn"}}
   {{output "ec2_create_trigger_db_update_lambda_arn" logicalId="EC2CreateTriggerDbUpdateLambdaArn"}}
+  {{output "ec2_create_trigger_db_update_lambda_proxy_arn" logicalId="EC2CreateTriggerDbUpdateLambdaProxyArn"}}
```
Comment (RajkumarShivage) [inline awsconfigrules/build.f/particles-awsconfigrules/particles/cftemplates/template.json.hbs]: Rename output to ec2_create_trigger_master_lambda_proxy_arn
Rename logicalId to EC2CreateTriggerMasterLambdaProxyArn

File: awsconfigrules/build.f/particles-awsconfigrules/particles/outputs/ec2_create_trigger_db_update_lambda_proxy_arn.hbs [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{{#output}}
+  "Value" : { "Fn::GetAtt" : [ "EC2CreateTriggerDbUpdateLambdaProxy", "Arn" ]}
```
Comment (RajkumarShivage) [inline awsconfigrules/build.f/particles-awsconfigrules/particles/outputs/ec2_create_trigger_db_update_lambda_proxy_arn.hbs]: Rename EC2CreateTriggerDbUpdateLambdaProxy to EC2CreateTriggerMasterLambdaProxy

File: awsconfigrules/index_ec2_create_trigger_db_update_proxy.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+});
+
+
+// This is the handler that's invoked by Lambda
+// Most of this code is boilerplate; use as is
+exports.handler = (event, context, callback) => {
+    var params = {
+        ClientContext: "ProxyLambda",
+        FunctionName: "awsconfigrules-ec2-create-trigger-db-update",
+        InvocationType: "Event",
```
Comment (RajkumarShivage) [inline awsconfigrules/index_ec2_create_trigger_db_update_proxy.js]: This is an async invocation, how do we handle if master lambda fails to add/insert the data in DB?

File: awsconfigrules/build.f/particles-awsconfigrules/particles/cftemplates/template.json.hbs (hunk 1 above)
Comment (jophine-antony) [inline awsconfigrules/build.f/particles-awsconfigrules/particles/cftemplates/template.json.hbs]: Done

File: awsconfigrules/build.f/particles-awsconfigrules/particles/cftemplates/template.json.hbs (hunk 2 above)
Comment (jophine-antony) [inline awsconfigrules/build.f/particles-awsconfigrules/particles/cftemplates/template.json.hbs]: Done

File: awsconfigrules/build.f/particles-awsconfigrules/particles/outputs/ec2_create_trigger_db_update_lambda_proxy_arn.hbs (hunk 3 above)
Comment (jophine-antony) [inline awsconfigrules/build.f/particles-awsconfigrules/particles/outputs/ec2_create_trigger_db_update_lambda_proxy_arn.hbs]: Done

File: awsconfigrules/index_ec2_create_trigger_db_update_proxy.js (hunk 4 above)
Comment (jophine-antony) [inline awsconfigrules/index_ec2_create_trigger_db_update_proxy.js]: Lambda function has a hard timeout of 5min. Hence, we have to implement queuing mechanism to fix this.

**Comment by RajkumarShivage** [comment]
LGTM

**Pull request merged**